src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -104,8 +104,12 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
               <ShowArtworksEmptyState show={show} />
             </>
           )}
-          <Separator as="hr" my={6} />
-          <ShowContextCard show={show} />
+          {show.fair?.hasFullFeature !== false && (
```

Comment:
**minor / non-blocking**: I think a truthy conditional would work just as well here:

```suggestion
          {show.fair?.hasFullFeature && (
```

Reply:
That was intended. Fair may be either object with `hasFullFeature` prop or `null`.
For instance here https://www.artsy.net/show/loft-art-gallery-mous-lamrabat-digital-solo-show?sort=partner_show_position fair is `null` (hence `hasFullFeature` is `falsy`) but link is not broken, it just refers to _/loft-art-gallery_, not to _/fair/loft-art-gallery_

Reply:
Ahh, that makes sense. Thanks for clarifying!!